contrib/Makefile.am

```diff
@@ -7,6 +7,6 @@ endif
 if PAM_ZFS_ENABLED
 SUBDIRS += pam_zfs_key
 endif
-DIST_SUBDIRS = bash_completion.d bpftrace dracut initramfs pam_zfs_key pyzfs zcp
+DIST_SUBDIRS = bash_completion.d bpftrace dracut $(ZFS_INITRAMFS) pam_zfs_key pyzfs zcp
```

Comment:
If I understand what you're going for we need to handle this in the same way as `PAM_ZFS_ENABLED`.  Specifically we need to always include `initramfs` in `DIST_SUBDIRS` so these sources are included in the `make dist` tarball, but only add it to `SUBDIRS` when support for initramfs is enabled.  Something like this, you'll need to update the configure bits accordingly of course.  Aside from that this looks good.

```diff
 SUBDIRS = bash_completion.d pyzfs zcp
 if BUILD_LINUX
-SUBDIRS += bpftrace dracut initramfs
+SUBDIRS += bpftrace dracut
+if INITRAMFS_ENABLED
+SUBDIRS +=  initramfs
+endif
 endif
 if PAM_ZFS_ENABLED
 SUBDIRS += pam_zfs_key
 ```

Reply:
changed.